content/log/explore.md

```diff
@@ -0,0 +1,150 @@
+
+### View a log line in context
+When displaying the detail of a log line, ask to « view in context ». 
+It removes any filters from your search bar and display log lines preceding this line and the few log lines following it.
+
+If you send the log lines in syslog format, the hostname and app name corresponding to the log line are automatically be selected.
+
+### Columns
+In order to get more details about the objects you are looking at, select a few facets to display with the **Columns** button:
+{{< img src="log/explore/log_list_with_columns.png" alt="Log List with columns" >}}
+
+## Facets 
+
+A facet displays all the distinct members of an attribute or a tag as well as provides some basic analytics such as the amount of logs represented. This is also a handle to easily filter over your data.
+
+A Facet helps you to break down your datasets over user(s), service(s), etc ...
+
+/////\\\\\\<br>
+/////\\\\\\<br>
+/////\\\\\\Add gif of how facet works<br>
+/////\\\\\\<br>
+/////\\\\\\<br>
+### Create a Facet
+
+To start using an attribute or a tag in a Facet or in the search, you simply need to click on it and add it as a Facet:
+{{< img src="log/explore/create_facet.png" alt="Create Facet" style="width:75%;">}}
+
+Once this is done, the value of this attribute for all new logs is  stored and can be used for search either in the search bar or in the Facet Panel
+
+### Facet Panel
+
+Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
+{{< img src="log/explore/facet_panel.png" alt="Facet panel" >}}
+
+### Expanded view
+
+Expand a facet to access advanced search options like 
+Including or excluding a pattern within a facet.
+
+For example: You want to have all host starting by “i-06”: 
+{{< img src="log/explore/expanded_view.png" alt="Expanded view" style="width:75%;" >}}
+
+## Search bar
+
+The search query language is based on the Lucene query string:
+
+[Apache Lucene - Query Parser Syntax](http://lucene.apache.org/core/2_9_4/queryparsersyntax.html)
+
+All search parameters are contained in the url, so it is very simple to share your view.
+
+### Search syntax
+A query is broken up into terms and operators.
+
+There are two types of terms:
+
+* A **Single Term** is a single word such as "test" or "hello".
+
+* A **Sequence** is a group of words surrounded by double quotes such as "hello dolly" or "hello dolly"~2.
+
+Multiple terms can be combined together with Boolean operators to form a more complex query.
+
+Here are the available boolean operators:
+
+||||
+|:----|:----|:----|
+| **Operator** | **Description ** | **Example **|
+| `AND` | **Intersection**: both terms are in the selected events (if nothing is added, AND is taken by default) | authentication AND failure |
+| `OR` | **Union**: either terms is contained in the selected events| authentication OR password|
+| `-` | **Exclusion**: the following term is NOT in the event |authentication AND -password|
+
```

Comment:
I think we should add a GIF for the auto complete functionality. The one Emmanuel used in the global email is just fine.

Reply:
👍

Reply:
GIF not working but i guess this is because of the beta version?

Reply:
gif is working.